pdns/rec_channel_rec.cc

```diff
@@ -935,6 +935,11 @@ string RecursorControlParser::getAnswer(const string& question, RecursorControlP
   }
 
   if(cmd=="reload-acls") {
+    if(!::arg()["chroot"].empty()) {
+      L<<Logger::Error<<"Unable to reload ACL when chroot()'ed, requested via control channel"<<endl;
```

Comment:
There is a deeper underlying problem, which is that -any- problem with accessing the config files will cause us to fully mess up the ACLs. We are mitigating it for the chroot situation here.